### ISSUE TITLE
Using -DKERNEL_PATH=<path/to/kernel> does not work

### ISSUE THREAD

**Issue by guidoj** [seL4/seL4_tools#141]
The find_file command on line 32 in cmake-tool/base.cmake searches explicitly for a directory named "kernel" and the result is stored in KERNEL_PATH. Renaming the kernel directory as stated in [Incorporating into your project](https://docs.sel4.systems/projects/buildsystem/incorporating.html) and defining KERNEL_PATH with the -D option will result in a bogus value for KERNEL_PATH resulting in cmake failing on line 40 (add_subdirectory). Also, the add_subdirectory command itself forces a fixed binary directory named "kernel", which seems unnecessary.

**Comment by axel-h** [comment]
Indeed, this is broken. Are you using this feature - and could you propose a patch to fix this?

**Comment by guidoj** [comment]
Sorry, but I am no longer using this feature. I guessed the solution was to have separate variables for the KERNEL_PATH parameter and the internal version, but that also affected the KERNEL_HELPERS_PATH variable. As I only used the seL4_tools to get me started, I did not pursue this any longer.
